docs/designers-developers/developers/filters/i18n-filters.md

```diff
@@ -0,0 +1,109 @@
+# i18n Filters
+
+The i18n functions (`__()`, `_x()`, `_n()` and `_nx()`) provide translations of strings for use in your code. The values returned by these functions are filterable if you need to override them, using the following filters:
+
+- `i18n.gettext`
+- `i18n.gettext_with_context`
+- `i18n.ngettext`
+- `i18n.ngettext_with_context`
+
+## Filter Arguments
+
+The filters are passed the following arguments, in line with their PHP equivalents.
```

Comment:
It would be nice to link with the corresponding PHP documentation page.

Reply:
I'm not aware of a single place we could link to that covers all of the relevant PHP filters (they each have their own page), so decided not to add links.